config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -1791,25 +1809,34 @@ end subroutine ModelAdvance
 
 
 subroutine ModelSetRunClock(gcomp, rc)
+
+  use ESMF, only : ESMF_TimeIntervalSet
```

Comment:
Do we need this line?

Reply:
Yes, it is not used anywhere else and we specify only used ESMF procedures.